source/guide_roundcube.rst

```diff
@@ -0,0 +1,222 @@
+.. highlight:: console
+
+.. author:: Achim | pxlfrk <hallo@pxlfrk.de>
+
+.. tag:: lang-php
+.. tag:: web
+.. tag:: mail
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/roundcube.svg
+      :align: center
+
+#################
+Roundcube Webmail
+#################
+
+.. tag_list::
+
+Roundcube webmail is a browser-based IMAP client with an easy-to-use user interface. It provides full functionality you expect from an email client, including MIME support, address book, folder manipulation, message searching and spell checking.
+
+The global Uberspace Webmail-Instance is also based on roundcube. Deploying your own roundcube webmail is ideal if you want to add functionality by plugins or customize the theme etc.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`PHP <lang-php>`
+  * :manual:`domains <web-domains>`
+  * :manual:`Mail <mail-access>`
+
+License
+=======
+
+Roundcube Webmail is released under the GNU General Public License_ version 3 or any later version with exceptions for skins and plugins.
+
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the stable version 7.3:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version use php 7.3
+ Selected PHP version 7.3
+ The new configuration is adapted immediately. Patch updates will be applied automatically.
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.3'
+ [isabell@stardust ~]$
+
+If you want to use your Roundcube Webmail with your own domain you need to setup your domain first:
+
+::
+
+ [isabell@stardust ~]$ uberspace web domain list
+ isabell.uber.space
+ [isabell@stardust ~]$
+ 
+You'll need your MySQL :manual_anchor:`credentials <database-mysql.html#login-credentials>`. Get them with ``my_print_defaults``:
+
+::
+
+ [isabell@stardust ~]$ my_print_defaults client
+ --default-character-set=utf8mb4
+ --user=isabell
+ --password=MySuperSecretPassword
+ [isabell@stardust ~]$
+ 
+
+
+
+Installation
+============
+
+``cd`` to your :manual:`document root <web-documentroot>`, download the latest release of Roundcube and extract it. Move all containing files up one level and remove the emtpty directory as well as the archive afterwards.
+
+.. note:: Check the Roundcube_ website or `Github Repository`_ for the latest stable release and copy the download link to the Complete.tar.gz file. Then use ``wget`` to download it. Replace the URL with the one you just got from GitHub/Website.
+
+.. code-block:: console
+ :emphasize-lines: 2,4,5,6,7,8
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ wget https://github.com/roundcube/roundcubemail/releases/download/1.4.3/roundcubemail-1.4.3-complete.tar.gz
+ [isabell@stardust html]$ tar xfz *.tar.gz
```

Comment:
You can use `tar --strip-components=1` to get rid of the `cd; mv; cd; rm` below. That way all the contents of `roundcubemail-1.4.3` inside the tar will end up directly in the current directory.

Reply:
I changed it to your suggestion, but couln't get rid of the final `rm -r *.tar.gz`. Initially i wanted to use curl, but i didn't get it to work properly. Thanks for any further suggestion to optimize.

Reply:
the last rm is fine :) thanks!